docs/source/eventproc.rst

```diff
@@ -0,0 +1,201 @@
+Event Processing
+=========================
+
+Overview
+--------
+
+The Event Processing library allows you to run arbitrary code (event handlers) at predefined events (also called hooks) within the PredictiveGrid platform.  A sample use case might be if you would like to upload new COMTRADE files using the COMTRADE ingress and have a transform or analytical process run immediately after import.
+
+Notifications
+~~~~~~~~~~~~~~~~~~~
+
+After your event handler is executed, an email is sent to the success or failure email address supplied during registration.  If successful, the email will contain any text that was returned by your Python function.  If an exception is raised, then the handler code is regarded as unsuccesful and a notification is sent to the corresponding failure email address with exception details.
+
+Dependencies
+~~~~~~~~~~~~~~~~~~~
+
+Our platform will provide a base python 3.7.3 environment with which you can add any desired libraries installable with `pip`.  When registering an event handler you can provide the content for a requirements file as if you had run `pip freeze` at the command line.  We provide some standard libraries such as:
```

Comment:
Again, as far as pip additions, I thought we were punting to v2.

Reply:
I must have been confused by this as I put in the code and docs.  I'm gonna remove it in favor of v2.